ERCS/erc-6860.md

```diff
@@ -293,8 +304,11 @@ contractName    = address
                 / domainName
 chainid         = %x31-39 *DIGIT
 
-pathQuery       = mPathQuery ; path+query for manual mode
-                / aPathQuery ; path+query for auto mode
+pathQuery       = mPathQuery [ evmQuery ] ; path+query for manual mode
+                / aPathQuery [ evmQuery ] ; path+query for auto mode
+
+evmQuery        = *( "&" blockNumberQuery )
+blockNumberQuery = "_blockNumber=" 1*DIGIT
```

Comment:
It will be great to add other `eth_call` parameters such as `gasLimit`.

Reply:
I added gasLimit and from, maybe more can be added